ImGui-based console (#1748)
f3d-app/f3d: doc/user/LIMITATIONS_AND_TROUBLESHOOTING.md

Your previous F3D installation is current being used to generate thumbnails, wait for a bit or reboot your computer before updating again.

> When running F3D without interactivity (e.g. `f3d.exe --version`) from CLI, nothing is displayed.

It's a limitation from the Windows "subsystem". You can workaround the issue by piping to a file (`f3d.exe --version > file.txt`) and read the file.

> Another solution is recompiling F3D with the CMake option `F3D_WINDOWS_GUI` disabled.

Reviewer: mention `git bash` maybe ?

> When running F3D without interactivity (e.g. `f3d.exe --version`) from CLI, nothing is displayed.

It's a limitation from the Windows "subsystem".

> You can workaround the issue by piping to a file (`f3d.exe --version > file.txt`) and read the file.

Reviewer: ```suggestion
You can workaround the issue by redirecting output to a file and printing the file:
`f3d.exe --version > file.txt` && cat file.txt`
```

Please make sure `cat` works in cmd though :)

Author: it doesn't

Reviewer: well it would be nice to find a command that works because we will need to update the issue template accordingly. I will take care of it

> We found that using git bash console works fine though.

Reviewer: ```suggestion
It also seems to be working fine with (git bash console)[https://gitforwindows.org/].
```
